src/Michalmanko/Holiday/Test/ProviderTest.php

```diff
@@ -47,6 +47,16 @@ public function testIsHoliday()
         $this->assertTrue($provider->isHoliday(new \DateTime('2014-01-06')));
         $this->assertTrue($provider->isHoliday(new \DateTime('2014-01-06'), Holiday\Holiday::TYPE_HOLIDAY));
         $this->assertFalse($provider->isHoliday(new \DateTime('2014-01-06'), Holiday\Holiday::TYPE_NOTABLE));
+
+    }
+
+    public function testDontModifyDates()
```

Comment:
You're testing only line 163. Could you add a test also for the line 151? Thanks!